readme.md

```diff
@@ -55,18 +109,26 @@ Post upload, the bucket contents should look similar to this:
 2019-09-30 14:14:36       1244 scripts/ssm
 ```
 
-It is recommended that this template be deployed in the same subnet as your EMR cluster.
 
 ### hail-ami
 
+This template is deployed _once_.
```

Comment:
Consider consolidating these paragraphs since this template is optional.   It may even be better to push the public AMIs here first, and this as a "if you like to brew your own".